contact.html

```diff
@@ -6,38 +6,40 @@
 	<meta charset="utf-8">
 	<meta name="description" content="Contact page for JumboCode website">
 	<meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel='stylesheet' href='style.css'/>
 
 	<!-- Favicon -->
 	<!-- <link rel="icon" href="/assets/img/favicon.ico" type="image/x-icon" /> -->
 
-	<!-- Stylesheets -->
```

Comment:
Please keep this, and put the `<link>` to the stylesheet below this comment - it's a good way to standardize our code